.addon

```diff
@@ -27,6 +27,10 @@
     "LeaderboardType": "None",
     "ProjectTemplate": null,
     "CsProjName": "",
-    "HttpAllowList": null
+    "HttpAllowList": [
```

Comment:
Is the HTTP allow list truly required now that Garry fixed the "data" scheme bug? Media seems to be playing fine on main, which has HttpAllowList set to null.